package.json

```diff
@@ -105,7 +105,7 @@
   ],
   "dependencies": {
     "@ape-egg/tailwind-rows-columns": "^1.0.2",
-    "@sesamy/sesamy-js": "^1.22.0",
+    "@sesamy/sesamy-js": "^1.22.9",
```

Comment:
We added a hasAccess helper function in this version as we kept writing the same code over and over again.